src/main/webapp/app/admin/user-management/user-management.component.html

```diff
@@ -176,6 +176,7 @@ <h2>
                                 <fa-icon [icon]="faWrench"></fa-icon>
                                 <span class="d-none d-md-inline" jhiTranslate="entity.action.edit">Edit</span>
                             </a>
+                            <button class="btn btn-success btn-sm" [disabled]="!isLdapProfileActive" (click)="ldapSync(user.login!)" jhiTranslate="entity.action.ldap"></button>
```

Comment:
I suggest, as Stephan mentioned, that the button should not be displayed, not just disabled.

Reply:
@Dominik-Weinzierl , I changed the button accordingly